rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -278,8 +363,10 @@ def _add_single_batch(self, item: SampleBatchType, **kwargs) -> None:
         else:
             self._next_idx += 1
 
-    @DeveloperAPI
-    def sample(self, num_items: int, **kwargs) -> Optional[SampleBatchType]:
+    @override(ReplayBufferInterface)
+    def sample(
+        self, num_items: Optional[int] = None, **kwargs
+    ) -> Optional[SampleBatchType]:
         """Samples `num_items` items from this buffer.
```

Comment:
This docstring is now duplicated, we can remove it here and leave it in the interface.

Reply:
fixed. I left this larger docstring in the `ReplayBuffer` b/c it's already quite specific to the `ReplayBuffer` implementation. The new `ReplayBufferInterface`is more generic.